src/Core/Components/Label/FluentInputLabel.razor

```diff
@@ -1,15 +1,15 @@
-@namespace Microsoft.FluentUI.AspNetCore.Components
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
 @inject LibraryConfiguration LibraryConfiguration
 
 
 @if (!string.IsNullOrEmpty(Label) || ChildContent is not null)
 {
-    <label for="@ForId" class="fluent-input-label" @attributes="@AdditionalAttributes" aria-label="@AriaLabel">
+    <label for="@ForId" class="@ClassValue" @attributes="@AdditionalAttributes" aria-label="@AriaLabel">
```

Comment:
That could be better to keep `fluent-input-label` and to use a new HTML attribute `orientation`. 

For the future, that will be easier to configure other styles.

Reply:
Done!